### ISSUE TITLE
Content Packs break when Cobblemon is installed

### ISSUE THREAD

**Issue by DanDragonWolf** [FlansMods/FlansModReloaded#59]
I was trying to use Create: Love and War with Cobblemon installed, and the game crashed. To make sure it wasn't just an error with Create: Love and War, I removed it, updated to the latest version of Flan's Mod: Reloaded, and installed the Vender's Game Content Pack, and was able to get into the game. But when I went into the creative menu and tried to go to the tabs with guns on them, the game crashed.

**Comment by DanDragonWolf** [comment]
[crash-2024-03-22_18.50.39-client.txt](https://github.com/FlansMods/FlansModReloaded/files/14729156/crash-2024-03-22_18.50.39-client.txt)

**Comment by PaxiTheAlchemist** [comment]
I have the exact same problem when testing with both [Vender's Game](https://www.curseforge.com/minecraft/mc-mods/venders-game-a-flans-mod-content-pack) (`2.0.193`) and [Create: Love and War](https://modrinth.com/mod/create-love-war) (`0.2-1.20.1`) with [Cobblemon](https://modrinth.com/mod/cobblemon) (`1.5.2+1.20.1`). When the player attempts to enter into the creative menu and navigate to the pages including any flan's dependant mods, the game freezes and crashes. Additionally, spawning in items using the `/give` command from these mods will also result in a crash. All tests used [FlansModReloaded](https://github.com/FlansMods/FlansModReloaded) version `0.4` to which all active mods were at their latest releases as of this post

Here are the logs of both mods separately with any required dependencies of each where needed:
[Create: Love and War
](https://mclo.gs/o0eFb0E)[Vender's Game](https://mclo.gs/oOksOmg)

Was this issue resolved for another version?

**Comment by PaxiTheAlchemist** [comment]
Upon further testing, I had some instances where I would manage to get past crashing when rendering any Flan's dependent mod content. With this I was able to navigate through the creative menu and give myself items, however, upon firing anything, the game would immediately crash.

I did manage to figure out a temporary solution that was able to get the game to not crash on viewing a flan dependent mod item or firing of any weapons. By modifying resource packs right after loading up Minecraft, where the enabling, disabling, or reordering of resource packs is done to cause minecraft to put up a loading screen while it loaded resources, the game doesn't crash at the usual problematic events. This solution only lasts for the game session to which restarting the client will cause the issue to come back

Here is a [log](https://mclo.gs/6yjcam9) from this test where this temporary solution is applied
